tokens.xml

```diff
@@ -8533,13 +8564,15 @@ This creature can't be enchanted.</text>
             <reverse-related>Tempting Contract</reverse-related>
             <reverse-related>Thieves' Tools</reverse-related>
             <reverse-related>Tireless Provisioner</reverse-related>
+            <reverse-related count="x">Tivit, Seller of Secrets</reverse-related>
```

Comment:
interestingly, because neither of these is excluded and both require x, the client does nothing at all when you try to use the create all tokens action

Reply:
Well that is awkward (and should maybe be raised as an issue on the cockatrice github). I suppose just making one or the other exclude is best.

Reply:
There has never been a card like this one before! Ouch.